src/calculateRentalCost.js

```diff
@@ -4,7 +4,19 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const carCostPerDay = 40;
+  const longTermDiskont = 50;
+  const shortTermDiskont = 20;
+  const longTerm = 7;
+  const shortTerm = 3;
+
+  if (days >= longTerm) {
+    return carCostPerDay * days - longTermDiskont;
+  } else if (days >= shortTerm) {
+    return carCostPerDay * days - shortTermDiskont;
```

Comment:
The use of 'else if' after a return statement is unnecessary. You can remove the 'else' to simplify the control flow as per the checklist item 5.